lib/forkhelpers.ml

```diff
@@ -23,6 +23,10 @@
 
 let default_path = [ "/sbin"; "/usr/sbin"; "/bin"; "/usr/bin" ]
 
+let test_path = Option.bind (Sys.getenv_opt "FE_TEST") (fun _ -> (Sys.getenv_opt "XDG_RUNTIME_DIR"))
```

Comment:
What does this code do? 
```
let bind o f = match o with None -> None | Some v -> f v
```
When `FE_TEST` is found, `XDG_RUNTIME_DIR` is used. But when it is not found, `test_path` is `None` and the code below turns this into `/var`. Hmm.

Reply:
That's the intended way to not break existing users of the forkexec library. It only makes sense to use `XDG_RUNTIME_DIR` when being tested by a developer (or in the CI).

Reply:
I've added a comment in the code to avoid confusing readers.